src/ProjectTemplates/BlazorTemplates.Tests/BlazorServerTemplateTest.cs

```diff
@@ -2,37 +2,44 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Collections.Generic;
 using System.Linq;
 using System.Net;
-using System.Threading;
+using System.Runtime.InteropServices;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.E2ETesting;
-using Microsoft.AspNetCore.Testing;
-using OpenQA.Selenium;
+using Microsoft.AspNetCore.BrowserTesting;
+using PlaywrightSharp;
+using ProjectTemplates.Tests.Infrastructure;
 using Templates.Test.Helpers;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace Templates.Test
 {
-    public class BlazorServerTemplateTest : BrowserTestBase
+    public class BlazorServerTemplateTest : BlazorTemplateTest
     {
-        public BlazorServerTemplateTest(ProjectFactoryFixture projectFactory, BrowserFixture browserFixture, ITestOutputHelper output) : base(browserFixture, output)
+        public BlazorServerTemplateTest(ProjectFactoryFixture projectFactory, PlaywrightFixture<BlazorServerTemplateTest> fixture, ITestOutputHelper output)
+            : base(fixture)
         {
-            ProjectFactory = projectFactory;
+            ProjectFactory = projectFactory; ;
+            Output = output;
+            BrowserContextInfo = new ContextInformation(CreateFactory(output));
         }
 
         public ProjectFactoryFixture ProjectFactory { get; set; }
-
+        public ITestOutputHelper Output { get; }
+        public ContextInformation BrowserContextInfo { get; }
         public Project Project { get; private set; }
 
-        [Fact]
-        public async Task BlazorServerTemplateWorks_NoAuth()
+
+        [Theory]
+        [InlineData(BrowserKind.Chromium)]
```

Comment:
Are we limited to setting which browsers to test on on a per-scenario basis or can we apply this to an entire class/project?

Reply:
It's an xUnit question, I don't have a clear answer. It might be there's a way to do this at the class level, but i'm not sure.

Reply:
https://andrewlock.net/creating-parameterised-tests-in-xunit-with-inlinedata-classdata-and-memberdata/#using-a-dedicated-data-class-with-classdata-

Reply:
I think what @captainsafia  was suggesting was to apply `[Theory] and `[InlineData]` to the class

Reply:
`[ClassData]` or `[MemberData]` are the class-level equivalents to `[InlineData]`, aren't they?